GameEngine/robot_model.py

```diff
@@ -91,6 +92,14 @@ def get_position_components_wcs(self) -> (float, float):
         """
         return self._convert_EFCS_to_field_CS(self._x_pos_EFCS, self._y_pos_EFCS)
 
+    def get_point_angle_wcs(self):
+
+        return self.pointing_angle
```

Comment:
the self.pointing_angle is in EFCS (find description in the RobotModel. __init__ method)

so if you require the pointing in EFCS you can access this parameter directly: 
`my_robot = RobotBasicModel()
angle = my_robot.pointing_angle`
 (WCS stands for World Coordinate System, the CS shared by all non players related components)

Anyway - keep this function for WCS - but take into consideration the `self._cord_system_rot` parameter

https://github.com/szgula/UofG_Robotics_TDP/blob/0dd4c3637a14d72422c78d8c6a44a278d3da4969/GameEngine/robot_model.py#L24